src/ui/components/Explorer.js

```diff
@@ -36,6 +55,9 @@ export type ExplorerState = {|
   showWeightConfig: boolean,
   view: CredView,
   recalculating: boolean,
+  anchorEl: any,
+  selectedIndex: number,
```

Comment:
What are these for?

Reply:
those are to handle the dropdown menu selection. I can rename and properly type anchorEl